src/rules/no_delete_var.rs

```diff
@@ -31,6 +31,33 @@ impl LintRule for NoDeleteVar {
     let mut visitor = NoDeleteVarVisitor::new(context);
     visitor.visit_module(module, module);
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows the deletion of variables
+
+`delete` is used to remove a property from an object.  Variables declared via
+`var`, `let` and `const` cannot be deleted (`delete` will return false).  Setting
+`strict` mode on will raise a syntax error when attempting to delete a variable.
+    
+### Invalid:
+```typescript
+const a = 1;
+let b = 2;
+var c = 3;
+delete a; // would return false
+delete b; // would return false
+delete c; // would return false
+```
+
+### Valid:
+```typescript
+var obj = {
+  a: 1
+}
```

Comment:
```suggestion
var obj = {
  a: 1,
};
```

`deno fmt` recommends semicolon and trailing comma.